tests/integration/test_pairwise_significance.py

```diff
@@ -571,6 +571,27 @@ def test_pairwise_cat_x_mr_gender_x_all_pets_owned(self):
             ),
         )
 
+    def test_pairwise_significance_indices(self):
+        transforms = {"pairwise_indices": {"alpha": [0.05, 0.08]}}
+        slice_ = Cube(
+            OL.CAT_X_MR_GENDER_X_ALL_PETS_OWNED, transforms=transforms
+        ).partitions[0]
+
+        assert slice_.column_percentages.tolist() == [
+            [75.0, 20.0, 60.0],
+            [25.0, 80.0, 40.0],
+        ]
+
+        assert slice_.pairwise_indices.tolist() == [
+            [(), (0, 2), ()],
+            [(1,), (), (1,)],
+        ]
+
+        assert slice_.pairwise_indices_alt.tolist() == [
+            [(), (0, 2), ()],
+            [(1,), (), (1,)],
+        ]
```

Comment:
could you use a sig level alt like 0.001 to see if the indices are different? Just to be sure that that method is doing something different

Reply:
done